src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    green,
```

Comment:
```suggestion
    GREEN,
```
enum values are written as constants